libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -780,13 +803,12 @@ void test_MQTT_Subscribe_Publish_With_Qos_2( void )
 
 void test_MQTT_Connect_LWT( void )
 {
-    int secondTcpSocket;
     NetworkContext_t secondNetworkContext = { 0 };
     bool sessionPresent;
     MQTTContext_t secondContext;
 
-    /* Establish a second TCP connection with the server endpoint, then
-     * a TLS session. The server info and credentials can be reused. */
+    /*Establish a second TCP connection with the server endpoint, then
```

Comment:
Is this due to uncrustify? It looks wrong.

Reply:
Fixed